Dockerfile-windows.template

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/windows/servercore:%%WINDOWS-VERSION%%
+
+SHELL ["cmd", "/S", "/C"]
+
+ENV RUST_VERSION=%%RUST-VERSION%%
+
+# Install the Visual Studio Build Tools.
+ADD https://aka.ms/vs/16/release/vs_buildtools.exe /vs_buildtools.exe
+RUN vs_buildtools.exe --quiet --wait --norestart --nocache \
+        --installPath C:\BuildTools \
+        --add Microsoft.Component.MSBuild \
+        --add Microsoft.VisualStudio.Component.Windows10SDK.%%SDK-BUILD%% \
+        --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 \
+    || IF "%ERRORLEVEL%"=="3010" EXIT 0
+RUN del vs_buildtools.exe
+RUN setx /M __VSCMD_ARG_NO_LOGO "1"
+
+ENTRYPOINT ["powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass"]
+
+# Install Rustup and Rust toolchains for x86 and x64
+ADD https://static.rust-lang.org/rustup/archive/%%RUSTUP-VERSION%%/x86_64-pc-windows-msvc/rustup-init.exe /rustup-init.exe
+RUN rustup-init.exe -y -v --default-toolchain %RUST_VERSION% --default-host x86_64-pc-windows-msvc
+RUN del rustup-init.exe
+RUN rustup toolchain install %RUST_VERSION%-i686-pc-windows-msvc
```

Comment:
Why install the 32 bit toolchain as well by default?

Reply:
My original use case requires both the 32 and 64 bit versions of the Rust toolchain for testing, so I installed both. If you would like, I can remove the 32 bit toolchain.

Reply:
Yeah I think we should leave it to just x86_64 in the base image. It's a quick command away if you need another target